LayoutTests/fast/dom/load-offsetWidth-expected.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
Well, ideally this wouldn't be empty and this would contain the "This test passes if it doesn't crash." text.